### ISSUE TITLE
[docker ouside of docker] Changing the workspace to `${localWorkspaceFolder}` on windows VS Code/Dev Containers does not call a workspace selector window

### ISSUE THREAD

**Issue by hellt** [devcontainers/features#1207]
Hi all,
While doing this in the devcontainer.json on macOS works great:

```json
    "workspaceFolder": "${localWorkspaceFolder}",
    "workspaceMount": "source=${localWorkspaceFolder},target=${localWorkspaceFolder},type=bind"
```

It does not work on windows 11 and vscode. Trying to run a devcontainer with this config does not open up a window with the workspace selection like it does on macOS, and instead the devcontainer build fails with invalid mount error calling out the workspace dir does not exist.

Is this a known issue?

**Comment by chrmarti** [comment]
Please append the Dev Containers log from when this happens. (`F1` > `Dev Containers: Show Container Log`)

I guess you had a multi-root workspace open on macOS to get the picker. `${localWorkspaceFolder}` does not show a picker.

The problem on Windows might be that you use `${localWorkspaceFolder}` as the target folder which needs to be a Linux path, but in this case is a Windows path.

**Comment by hellt** [comment]
@chrmarti yes, I am using the docker-outside-of-docker feature and follow the [provided recommendations](https://github.com/devcontainers/features/blob/main/src/docker-outside-of-docker/README.md#2-change-the-workspace-to-localworkspacefolder) to enable the binds.

When using the devcontainer on macOS (using the OrbStack as the docker engine provider) when I first launch the devcontainer I get a picker to select the workspace, and the picker uses the paths of the Linux VM.

On windows we do not get this picker, and judging by your comment it might be expected due to the Windows path elements?

**Comment by chrmarti** [comment]
Could you append a screenshot of the picker? I think it's the workspace picker for multi-root workspaces, the screenshot would allow to confirm this.

**Comment by hellt** [comment]
I struggle to invoke the same picker again locally... It appears only when running docker outside of docker with codespaces

<img width="575" alt="image" src="https://github.com/user-attachments/assets/db9b37c1-39c0-4486-98a9-9f315f25578d" />
<img width="636" alt="image" src="https://github.com/user-attachments/assets/ba14efc6-6218-40e3-ba3b-9704055ec175" />
